src/uct/ib/rc/accel/rc_mlx5_ep.c

```diff
@@ -670,20 +671,25 @@ ucs_status_t uct_rc_mlx5_ep_fc_ctrl(uct_ep_t *tl_ep, unsigned op,
 ucs_status_t uct_rc_mlx5_ep_get_address(uct_ep_h tl_ep, uct_ep_addr_t *addr)
 {
     UCT_RC_MLX5_EP_DECL(tl_ep, iface, ep);
-    uct_rc_mlx5_ep_flush_address_t *rc_addr = (uct_rc_mlx5_ep_flush_address_t*)addr;
+    uct_rc_mlx5_ep_address_t *rc_addr       = (uct_rc_mlx5_ep_address_t*)addr;
     uct_ib_md_t *md                         = uct_ib_iface_md(ucs_derived_of(
                                               tl_ep->iface, uct_ib_iface_t));
+    uct_rc_mlx5_ep_flush_address_t *flush_addr;
```

Comment:
uct_rc_mlx5_ep_ext_address_t 
since it's not flush any more. just added flags.

Reply:
right, renamed